### PR TITLE
Features/output

### PR DESCRIPTION
**Pull request by Stalfy**
New elements keys. Needs cleanup once merged with @Lolskate

### CODE REVIEW

File: src/output/generator.js [hunk 1]
```diff
@@ -16,14 +18,44 @@ const PARAMETER_NAME_INDEX = 2;
...
+const PARAMETER_ELEMENTS_REGEX = `\\[(.*)?\\]\\((?:${PARAMETER_DISCRIMINANTS_REGEX})\\:?\\s?([\\w\\d]+)\\)`;
+
+const ELEMENT_DISCRIMINANTS_REGEX = `${INVOKER_DISCRIMINANTS_REGEX}|${MODULE_DISCRIMINANTS_REGEX}|${PARAMETER_DISCRIMINANTS_REGEX}|${TEST_DISCRIMINANTS_REGEX}`;
+const ELEMENT_DETECTION_REGEX = `\\[.*?\\]\\((?:${ELEMENT_DISCRIMINANTS_REGEX})\\:?\\s?.*?\\)+`;
+
+const DISCRIMINANT_DETECTION_REGEX = `\\]\\((${ELEMENT_DISCRIMINANTS_REGEX})`;
 
 class HTMLGenerator {
     constructor() {
         this.elementFormattingFunctions = {};
```
Comment (Grasseh) [inline src/output/generator.js]: Would probably be cleaner to do:
```
this.elementFormattingFunctions = {
  i : this._formatInvoker.bind(this),
  inv = this._formatInvoker.bind(this)
}
```
etc

**Review by Grasseh — CHANGES_REQUESTED** [review]

File: src/output/generator.js (hunk 1 above)
Comment (Stalfy) [reply src/output/generator.js]: Applied requested changes. Left single quotes beause of # symbol and var reserved word.

**Review by Grasseh — APPROVED** [review]

**Review by KB07 — APPROVED** [review]
seems good to me.

**Review by Lolskate — APPROVED** [review]
Works like a charm, added the same logic in the parser

**Pull request merged**